CHANGELOG.md

```diff
@@ -1,5 +1,246 @@
+- Deleting marker cancel 2/N. [DO NOT LAND] ([c38318d084](https://github.com/facebook/react-native/commit/c38318d08469ad30535d970328704dc9ae6bb785) by [@mkareta](https://github.com/mkareta))
+- Support PerformanceLogger stopTimespan updates ([08c338eebf](https://github.com/facebook/react-native/commit/08c338eebf67ef6c8c8fb7e3a91bbf89bbc2bb4c) by [@sahrens](https://github.com/sahrens))
+- Added PlatformColor implementations for iOS and Android ([f4de45800f](https://github.com/facebook/react-native/commit/f4de45800f25930a1c70f757d12269d859066d3d) by [@tom-un](https://github.com/tom-un))
+- Stamp React Native Version Into C++ Code ([427ba359e0](https://github.com/facebook/react-native/commit/427ba359e0c9411438286dd137bbca67f9829fde) by [@NickGerleman](https://github.com/NickGerleman))
+- Add nightly CI build ([7d2daa0a6d](https://github.com/facebook/react-native/commit/7d2daa0a6d4b687678f8671de76c4a2a3cf2318a) by [@sweggersen](https://github.com/sweggersen))
+- New <Pressable> Component to make it easier to create touchable elements ([3212f7dfe8](https://github.com/facebook/react-native/commit/3212f7dfe82d187e27f1410c8c3cb1d9fb9f5094) by [@TheSavior](https://github.com/TheSavior), [bd3868643d](https://github.com/facebook/react-native/commit/bd3868643d29e93610e19312571a9736df2cbdf8) by [@vonovak](https://github.com/vonovak))
+- Export LogBox module ([799bf56f6f](https://github.com/facebook/react-native/commit/799bf56f6f6a46b6bd42ac5a824f44bd1412f3b6) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Export LayoutAnimationConfig flow type ([f0dafd34fe](https://github.com/facebook/react-native/commit/f0dafd34fedb0d63eb2499b978a52da9e6b71ea1) by [@sahrens](https://github.com/sahrens))
+- Added `react-native-community/eslint-plugin` as a dependency for `react-native-community/eslint-config` ([2c2e35c634](https://github.com/facebook/react-native/commit/2c2e35c634cd936bd7ea7a7fe444058268308224) by [@Naturalclar](https://github.com/Naturalclar))
+- DEBUG_NETWORK_SEND_DELAY can be used to simulate slow network. ([4aac019176](https://github.com/facebook/react-native/commit/4aac019176e3359068ac671ed4157a6e3ada481f) by [@sahrens](https://github.com/sahrens))
+- Support for accessibility Label prop to the Picker component ([0a525b6d9d](https://github.com/facebook/react-native/commit/0a525b6d9d2a88dddf24b85a2485b928fca23b16) by [@KevinGVargas](https://github.com/KevinGVargas))
+- Allow zIndex to useNativeDriver ([6a4e06faa8](https://github.com/facebook/react-native/commit/6a4e06faa8afbcb607fc2696c45c4f3257b6665d) by [@mackenziemance](https://github.com/mackenziemance))
+
+#### Android specific
+
+- Add analyticsTag prop into ImageView component ([72150f57ce](https://github.com/facebook/react-native/commit/72150f57ced3c626bb1c3a48a8609585c9a5a9d7), [0128e4602e](https://github.com/facebook/react-native/commit/0128e4602ecbe5ff771c1d1951e622b9c414180f) by [@mdvacca](https://github.com/mdvacca))
```

Comment:
This prop was renamed to `internal_analyticTag` and isn't meant for public consumption. Should be removed from the changelog